evaluate.py

```diff
@@ -78,6 +78,8 @@ def main(
         if not isinstance(dt, torch.dtype):
             raise ValueError(f"{dtype} is not a valid dtype.")
         dtype = dt
+    else:
```

Comment:
In L77 you can replace the default return `None` with `torch.float32` and remove this else block

This suggestion applies to the other files too